perf/types.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package perf
+
+import v1 "github.com/google/cadvisor/info/v1"
+
+type UncoreCollector interface {
```

Comment:
Consider using `stats.Collector` interface.

Reply:
`UncoreCollector` interface is more comfortable for me. I would like to get only `[]PerfStat`, nor create new `ContainerStats` and update it.

Reply:
I don't think that your justification is still relevant.